### PR TITLE
RHCLOUD-31713 Adding metrics for flow started/finished

### PR DESCRIPTION
**Pull request by josejulio**
Still need to add the metric to more flows or at least key flows that we want to start tracking.

### CODE REVIEW

File: common/metrics.py [hunk 1]
```diff
@@ -1,8 +1,37 @@
...
+    CLOSING = "closing"
+    CLOSING_ANYTHING_ELSE = "closing_anything_else"
+    FEEDBACK = "feedback"
+
+
+def flow_started_count(flow_name: Flow):
+    _flows_started_count.labels(flow_name=flow_name.value).inc()
+
+
+def flow_finished_count(flow_name: Flow, sub_flow_name: str = ""):
```
Comment (justinorringer) [inline common/metrics.py:34]: Looks good and easy to use to me!

Like Tahmid says you can try running it locally, but I think we can give it a go in stage.

**Review by justinorringer — APPROVED** [review]

File: common/metrics.py (hunk 1 above)
Comment (josejulio) [reply common/metrics.py:36]: > Like Tahmid says you can try running it locally, but I think we can give it a go in stage.

added a few more flows - we can try this on stage

**Review by tahmidefaz — APPROVED** [review]
I haven't tried it out locally, but the changes looks good to me. We can try it out in stage. :+1:

**Comment by josejulio** [comment]
:ship: it!

**Pull request merged**